src/Sylius/Bundle/PayumBundle/Resources/config/services.xml

```diff
@@ -33,6 +33,7 @@
 
         <!-- PayPal -->
         <service id="sylius.payum.action.convert_payment_to_paypal_express" class="Sylius\Bundle\PayumBundle\Action\ConvertPaymentToPaypalExpressAction">
+            <argument type="service" id="sylius.checkout.invoice_number_generator" />
```

Comment:
``sylius.invoice_number_generator`` is enough. It does not really belong to any "checkout" group of services.